README.rst

```diff
@@ -101,29 +104,41 @@ The current development branch of PyMC3 can be installed from GitHub, also using
     pip install git+https://github.com/pymc-devs/pymc3
 
 To ensure the development branch of Theano is installed alongside PyMC3
-(recommended), you can install PyMC3 using the ``requirements.txt``
-file. This requires cloning the repository to your computer:
+(recommended), see the Theano Installation section, below.
+
+Lastly, another option is to clone the repository and install PyMC3 using
+``python setup.py install`` or ``python setup.py develop``.
+
+Theano Installation
+-------------------
+
+PyMC3 is tested on Python 3.6, 3.7, and 3.8 and depends on `Theano-PyMC <https://github.com/pymc-devs/Theano-PyMC>`__,
+NumPy, SciPy, and pandas
+(see `requirements.txt <https://github.com/pymc-devs/pymc3/blob/master/requirements.txt>`__ for version
+information).
+
+That being said, you can install PyMC3 using the ``requirements.txt``
+file.
+
+This requires cloning the repository to your computer:
 
 ::
 
     git clone https://github.com/pymc-devs/pymc3
     cd pymc3
     pip install -r requirements.txt
 
-However, if a recent version of Theano has already been installed on
-your system, you can install PyMC3 directly from GitHub.
+.. note::
+   If you already have `Theano-PyMC <https://github.com/pymc-devs/Theano-PyMC>`__ installed, then re-install the module, as such
```

Comment:
We should say that it now requires Theano-PyMC instead of just the old Theano, and that if the old Theano is installed, they just remove that and install Theano-PyMC via conda-forge.

Reply:
@twiecki Sounds great, I think I made the appropriate changes. Moreover, I changed the sub-bulletin title of the `Installation` section to be `Theano-PyMC Installation`.

Let me know what you think.